### PR TITLE
Add configuration for LTS Prow

### PR DESCRIPTION
**Pull request by MaxHorstmann**

### CODE REVIEW

File: .github/workflows/deploy-lts-prow.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          resourceGroupName: ${{ vars.AZURE_RG }}
+          template: ./config/prow/cluster/prow-cluster.bicep
+          failOnStdErr: false
+
+      - name: Fetch config
+        run: |
+          echo "PROW_HOST=${{ steps.bicep.outputs.prowHostName }}" >> "$GITHUB_ENV"
+          echo "AZURE_STORAGE_ACCOUNT_USER=${{ steps.bicep.outputs.storageAccountName }}" >> "$GITHUB_ENV"
+
+      - name: Fetch storage key
```
Comment (haitch) [inline .github/workflows/deploy-lts-prow.yaml:37]: I was hopping this can get from bicep output directly

Comment (MaxHorstmann) [reply .github/workflows/deploy-lts-prow.yaml:37]: Not recommended: https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/linter-rule-outputs-should-not-contain-secrets

**Comment by haitch** [comment]
I think you miss projectcontour crd.

File: config/prow/cluster/prow-frontdoor.bicep [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  }
+}
+
+resource prowOrigin 'Microsoft.Cdn/profiles/origingroups/origins@2022-11-01-preview' = {
+  parent: prowOriginGroup
+  name: 'default-origin'
+  properties: {
+    hostName: ingressIP
+    httpPort: 80
+    httpsPort: 443
```
[the remark this answers is not in the document]
Comment (haitch) [reply config/prow/cluster/prow-frontdoor.bicep:50]: nevermind, on the EndpointToOrigin we have ```forwardingProtocol: 'HttpOnly'```

**Review by haitch — APPROVED** [review]

**Pull request merged**